omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
 
-Install PostgreSQL database server::
+Requirements
+------------
+
+1. Open a command-line terminal and install git if not already present::
+
+    $ brew install git
+
+2. Install PostgreSQL database server::
 
-    $ export LANG=${LANG:-en_US.UTF-8}
-    $ export LANGUAGE=${LANGUAGE:-en_US:en}
     $ brew install postgresql
 
-.. _`Homebrew and Python`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md
+   To ensure PostgreSQL uses UTF-8 encoding, open your bash profile and 
+   add the following enviornment variables::
+
+    export LANG=en_US.UTF-8
+    export LANGUAGE=en_US:en
+
+3. OMERO depends on Ice 3.6 and unfortunately does not run with 
+   the latest version of Ice at this time (Ice 3.7.3). To obtain 
+   Ice 3.6, we need to add a *tap* to Homebrew::
 
-You should install OMERO using Python 2.7 provided by
-Homebrew since it makes using Homebrew-provided modules
-simpler, for example the Ice python bindings needed by OMERO. For a
-more thorough description of the Homebrew solution, see the `Homebrew
-and Python`_ page. Note that the automated script linked above tests
-the OMERO installation using the Homebrew Python.
+    $ brew tap zeroc-ice/tap
+    $ brew install zeroc-ice/tap/ice36
 
-To install the Python provided by Homebrew::
+.. Note::
+   If you already have a version of Ice that is not 3.6 installed, you can instruct Homebrew to *unlink* it using 
+   ```$ brew unlink ice```. You can then instruct Homebrew to link to Ice 3.6 using ```$ brew link ice@36```
+
+4. Install Python provided by Homebrew::
 
     $ brew install python
 
-Check that Python is working and is version 2.7::
+   Homebrew installs Python in the following location::
+
+    '/usr/local/opt/python/libexec/bin'
+
+   Follow the instructions from the brew Python install and set the Homebrew version of Python 
+   to be used rather than the Python shipped with OS X. Add the following line to your :file:`.bash_profile`::
+
+    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+
+.. note:: 
+   To keep things a little cleaner, add the following enviornment variable to your :file:`.bash_profile`::
+
+    # Environment variable pointing to Homebrew Python location
+    export PYTHON_BREW=/usr/local/opt/python/libexec/bin
+
+   and append it to the :envvar:`PATH`::
+
+    export PATH="$PYTHON_BREW:$PATH"
+
+5. Check that Python is working and is version 2.7.x::
 
     $ which python
-    /usr/local/bin/python
+    /usr/local/opt/python/libexec/bin/python
+
     $ python --version
-    Python 2.7.9
+    Python 2.7.13
 
-The installation of OMERO via Homebrew depends upon two alternate
-repositories containing extra formulae:
-https://github.com/Homebrew/homebrew-science for the HDF5 formula and
-https://github.com/ome/homebrew-alt for all the OME-provided formulae
-and older versions of Ice. To add these, run::
+6. For developing with OMERO, or Python in general, we would recommend you use Virtualenv.
```

Comment:
``...we would recommend you use Virtualenv`` -> ``...we recommend the use of Virtualenv.``